mindsdb/integrations/handlers/langchain_handler/tools.py

```diff
@@ -138,12 +139,14 @@ def setup_tools(llm, model_kwargs, pred_args, executor, default_agent_tools, ope
             # user defined custom functions
             function_tools.append(tool)
 
-    skill_tools = []
+    tools = []
     skills = pred_args.get('skills', [])
     for skill in skills:
-        skill_tools += make_tools_from_skill(skill, llm, openai_api_key, executor)
+        tools += make_tools_from_skill(skill, llm, executor)
+
+    if len(tools) == 0:
```

Comment:
We need to document this change in behavior once merged.

Reply:
do you mean this item in PR description: 'don't use standard tools if skills are defined'?
@martyna-mindsdb, where is better to do it?